### PR TITLE
Deploy to heroku button

### PR DESCRIPTION
**Pull request by ilyaglow**
Hey there,

I believe Lair sadly doesn't receive enough love from community, so I decided to make another option (besides [handy docker compose](https://github.com/ryhanson/lair-docker)) to deploy it and check out how amazing it is.

With heroku you will need to install literally nothing to test the basics of Lair workflow, only a heroku account.

~~Btw I fixed a little mess with `babel-runtime` and `babel-compile` packages versions. The fix probably should be in a separated PR, ping me if you think so. The deploy button feature doesn't work without it, because obviously meteor fails to build the project properly.~~ 
_Update:_ 
The problem was on the `node` side, it didn't have `babel-runtime` to run meteor app build that uses babel which is weird.

As a bonus all other packages are updated.

You can try it on my [heroku-button branch](https://github.com/ilyaglow/lair/tree/heroku-button).

If everything is fine I'll point `app.json` and button in `README` to the base repository, but of course you can merge my changes manually.

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Lair",
+  "description": "Lair is a reactive attack collaboration framework and web application built with meteor",
+  "repository": "git://github.com/ilyaglow/lair.git#heroku-button",
```
Comment (tomsteele) [inline app.json]: Does this need to be updated to be /lair-framework/lair.git?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # lair
 Lair is a reactive attack collaboration framework and web application built with meteor.
 
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/ilyaglow/lair/tree/heroku-button)
```
Comment (tomsteele) [inline README.md]: Does this need to be /lair-framework/lair?

File: app.json (hunk 1 above)
Comment (ilyaglow) [inline app.json]: Yes, surely. I was just too lazy to add/manage an additional branch for the PR, so I can test it and stuff.

**Comment by ilyaglow** [comment]
@tomsteele done!

**Comment by ilyaglow** [comment]
Whoops! Heroku build log says that package versions constraints on the `standard-minifier-js` went wrong. Already sending the fix.

**Pull request merged**